.github/ISSUE_TEMPLATE/bug-performance-report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug/Performance report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**System information**
+- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
+- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
+- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
+- TensorFlow installed from (source or binary):
+- TensorFlow version (use command below):
+- Python version:
```

Comment:
Would be also good to find out if it Anaconda Python, Stock Python as in Mac, or homebrew installed Python etc.

Reply:
Added